sdk/src/Services/DynamoDBv2/Custom/DocumentModel/TableConfig.cs

```diff
@@ -30,6 +28,13 @@ public class TableConfig
         /// </summary>
         public string TableName { get; set; }
 
+        /// <summary>
+        /// The document API relies on an internal cache of the DynamoDB table's metadata to construct and validate 
+        /// requests. This controls how the cache key is derived, which influences when the SDK will call DescribeTable
```

Comment:
nit: add a doc ref to "DescribeTable"

Reply:
Addressed in https://github.com/aws/aws-sdk-net/compare/8a4f5a780d437debec1f38696b9dc2f069b991c1..1161dc30f02a5a18108c28b0191f21da5080a368